docs/plugins.rst

```diff
@@ -178,7 +178,7 @@ can use the HadoopSecurityManager to take care of security.
 
 For unsecure Hadoop cluster, there is nothing special that is needed.
 
-For secure Hadoop clusters, there are two ways inlcuded in the
+For secure Hadoop clusters, there are two ways included in the
```

Comment:
I don't see any change here.

Reply:
Spelling mistake. 
in**L**cuded
inc**L**uded